src/components/visualizations/chart/highcharts/commonConfiguration.ts

```diff
@@ -98,6 +99,12 @@ const BASE_TEMPLATE: any = {
         style: {
             fontFamily: 'Avenir, "Helvetica Neue", Arial, sans-serif',
         },
+        events: {
+            afterGetContainer() {
+                // allow tooltip over the container wrapper
```

Comment:
please consider `allow tooltip displays out of container element`.
furthermore, please separate function so it could be
```
1.
{
    events: {
        afterGetConainter: handleTooltipOffScreen
    }
}

or 
2.
{
    events: {
        afterGetConainter: () {
                handleTooltipOffScreen();
                // more handlers later here
        }
    }
}
```

I think 1. is enough for now :)

Reply:
is it really necessary? it's quite simple.

Reply:
Aliasing function name could improve readability.
`afterGetConainter` does not make sense with what we fix the issue.

Reply:
done